src/Task/PhpMd.php

```diff
@@ -58,12 +58,12 @@ public function run(ContextInterface $context): TaskResultInterface
         $extensions = $config['triggered_by'];
 
         $files = $context->getFiles();
-        if (0 !== count($whitelistPatterns)) {
+        if (0 !== \count($whitelistPatterns)) {
```

Comment:
Can you change this to `if (\count($whitelistPatterns)) {}`

Reply:
Done